### ISSUE TITLE
export csv only have issue id when i use export all colums

### ISSUE THREAD

**Issue by peney83** [maxtepkeev/python-redmine#233]
when i use the github master branch code and follow code  ,  and i want to export the issues to csv with all columns, but the exported csv only have the issue ID. please help me to check it 
```python
 issues = self.redmine.issue.filter(project_id='jsg', status_id='*')
issues.export('csv', savepath=self.export_path, filename=filename, columns="all")
```

**Comment by peney83** [comment]
I found the root cause. 
it modify the function "export" in  resultset.py 

```python
params.update({'c[]': columns, 'encoding': 'UTF-8'})
```

to 

```python
params.update({'**csv[columns]'**: columns, 'encoding': 'UTF-8'})
```

**Comment by peney83** [comment]
I found the root cause.
it modify the function "export" in resultset.py

```python
params.update({'c[]': columns, 'encoding': 'UTF-8'})
```
to
```python
            if columns == 'all':
                **columns = 'all'**

            params.update(**{'csv[columns]':** columns, 'encoding': 'UTF-8'})
```

**Comment by maxtepkeev** [comment]
@peney83 This is weird, the current code works for me, what Redmine version are you on ?

**Comment by peney83** [comment]
this is my redmine version 

Environment:
  Redmine version                3.3.0.stable
  Ruby version                   2.1.9-p490 (2016-03-30) [x86_64-linux]
  Rails version                  4.2.6
  Environment                    production
  Database adapter               Mysql2

**Comment by maxtepkeev** [comment]
Unfortunately I don't have that old Redmine, but I've checked 3.4.0 and my code works there, so they either fixed something between 3.3.0 and 3.4.0 or you have some 3rd party plugin installed which somehow influences your csv export functionality.

**Comment by maxtepkeev** [comment]
So it turned out export behaviour indeed changed in Redmine 3.4.0. I've implemented a backwards compatible behaviour in b79b7e34850ec49c390b1932051caa148f9e0f61. Be sure to set your Redmine version during Redmine object initialization otherwise it won't work.

**Comment by celes-mani** [comment]
Max, I'm on an even older version 1.3.1 When i don't use the columns attribute, i get the basic set of columns. When i use `columns="all"` I just get the 1st column, the id column. I want all the columns data, i'd appreciate if you could help here. Thx.
